docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+examples:
+
+```
+@my_data.h2o_temperature{location, state, bottom_degrees, surface_degrees, time > -3h}
+    |> select(fn: min("bottom_degrees"), by: [])
+```
+
+| location | state | bottom_degrees | surface_degrees | time
+| --- | --- | --- | --- | --- |
+| puget_sound | WA | 40 | 55 | 2020-02-22T17:10 |
+| puget_sound | WA | 40 | 54 | 2020-02-22T17:40 |
+
+Notice that all the columns are pulled through, the selected column name remains the same and note that 
+we've pulled in two records. We pull in two records because both of those have the same `min` `bottom_degrees` 
+value of `40`, which is the min for all records in that time. Notice also that `time` is the same value as those 
+individual records, which you don't see in calls to `aggregate`.
+
+You can also select in groups based on time or by other columns, just like you can with `aggregate`. Here's 
+an example using time grouping and inserting the time window for associated row selections:
+
+```
+@my_data.h2o_temperature{location, state, bottom_degrees, time > -3h}
+    |> selector(fn: max(bottom_degrees), by: ["state"], window: 1h, windowColumn: "window")
+
+// if you include window_column, the result will put the window timestamp for each selected row in that column
+```
+
+Produces this result:
+
+| location | state | bottom_degrees | time | window |
+| --- | --- | --- | --- | --- |
+| santa_monica | CA | 50 | 2020-02-22T15:01 | 2020-02-22T16:00 |
+| coyote_creek | CA | 51 | 2020-02-22T16:05 | 2020-02-22T17:00 |
+| santa_monica| CA | 48 | 2020-02-22T17:01 | 2020-02-22T18:00 |
+| santa_monica| CA | 48 | 2020-02-22T17:31 | 2020-02-22T18:00 |
+| santa_monica| CA | 48 | 2020-02-22T17:46 | 2020-02-22T18:00 |
+| puget_sound | WA | 40 | 2020-02-22T17:10 | 2020-02-22T18:00 |
+| puget_sound | WA | 40 | 2020-02-22T17:40 | 2020-02-22T18:00 |
+
+What this does is for each 1 hour period, it groups the rows by state and then selects the rows with the max value. 
+Because we included the `windowColumn` argument, that is included in the result with the ending timestamp of the 
+window that each record fell into.
+
+Since so many of the records have tied `max(bottom_degrees)` values, we see that many of the original 
+rows were selected in the result. We can also see the `window` that is inserted to specify 
+which block of time the selector was working across.
+
+The following functions can be used in the **Selector Predicate Block**: `first`, `last`, `min`, `max`, `top`, `bottom`
+
+#### Calling Aggregates without the Aggregate function
+The `aggregate` function provides a shorthand for computing multiple aggregates at a time. However, the 
+functions passed in can also be called on tables. For example:
+
+```
+// get the most recently written record
+@my_data.h2o_temperature |> last()
+
+// get the most recently written record for each location
+@my_data.h2o_temperature |> last(by: ["location"])
```

Comment:
Shouldn't you be able to do this without the last function since thats the default behavior? `@my_data.h2o_temperature{location, <some way to indicate all the fields>}